### PR TITLE
chore: Use env instead of custom parameter to webpack (#9934) (CP: 2.8)

### PR DESCRIPTION
**Pull request by Artur-**
This is compatible with Webpack 5

### CODE REVIEW

**Comment by caalador** [comment]
`liveReloadOnTouchedFrontendFile` is failing and I can't see why this change would have made a difference there.

**Comment by Artur-** [comment]
For the record: Live reload depends on the watchdog and using `--env` does not actually pass an environment variable to the webpack process. Not sure why that has ever worked (or maybe it has not...)

File: flow-server/src/test/java/com/vaadin/flow/server/frontend/FrontendUtilsTest.java [hunk 1]
```diff
@@ -244,14 +244,14 @@ public void commandToString_longCommand_resultIsWrapped() {
         List<String> command = Arrays.asList("./node/node",
                 "./node_modules/webpack-dev-server/bin/webpack-dev-server.js",
                 "--config", "./webpack.config.js", "--port 57799",
-                "--watchDogPort=57798", "-d", "--inline=false", "--progress",
+"--env watchDogPort=57798", "-d", "--inline=false", "--progress",
```
Comment (caalador) [inline flow-server/src/test/java/com/vaadin/flow/server/frontend/FrontendUtilsTest.java:247]: The test could just drop the watchDogPort all in all as it doesn't work.

Comment (Artur-) [reply flow-server/src/test/java/com/vaadin/flow/server/frontend/FrontendUtilsTest.java]: Done

**Review by caalador — APPROVED** [review]

**Pull request merged**